docs/src/pages/components/autocomplete/autocomplete.md

```diff
@@ -101,6 +101,14 @@ Head to the [customization](#customization) section for an example with the `Aut
 
 {{"demo": "pages/components/autocomplete/Asynchronous.js"}}
 
+If the logic is fetching new options at each keystroke,
+using the current value of the textbox to filter on the server,
+you need to disable the built-in filtering of the autocomplete component:
```

Comment:
```suggestion
If your logic is fetching new options on each keystroke and, 
using the current value of the textbox to filter on the server, 
you need to disable the built-in filtering of the autocomplete component:
```

Reply:
Line 106 needs the trailing comma the be removed (or possibly moved before "and".
Possibly "is using" for clarity.